### DISCUSSION TITLE
✅ 🚀 Feature Request: Make Nested Tabs always appear as Accordions in all breakpoints

### DISCUSSION THREAD

**Discussion by vgstef** [elementor/elementor#22915] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

I need a nested accordion. The nested tab widget can become a nested accordion on tablet and mobile. But there is no option to show it as a nested accordion on desktop.

That would be very easy for the delopper to add the option "All" in the breakpoint dropdown field to always show the nested tabs has an accordion.

### Describe the solution you'd like

Just add a "all" breakpoint option in the tab widget.

### Additional context

I've seen that Elementor is planning to develop a nested accordion widget. But an accordion widget is already there with the nested tab widget. It's just not active for desktop.

**Comment by nicholaszein** [comment]
Hello there @vgstef!

We have great news! 🙌 

### 📢 We're happy to announce that this feature was released with the new nested Accordion in Elementor v3.15! 🥳 

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
